.github/workflows/build.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         override: true
     - name: Check
       run: cargo clippy
+    - name: Format
+      run: cargo fmt --check
```

Comment:
[minor] separate `cargo fmt` to different PR maybe easier to review.

Reply:
I kept the changes in two separate commits so it's easier to review. Was planning to move to different MRs, but the catalog change has a dependency on the cargo fmt change because my editor automatically formats the file whenever I save it :D

Reply:
> I kept the changes in two separate commits so it's easier to review.

right, I miss it 😞 : https://github.com/roapi/roapi/pull/258/commits